src/components/TeamSection.svelte

```diff
@@ -17,24 +17,28 @@
   }
 </style>
 
-<Section>
+<Section id="about">
```

Comment:
I think it was called `anchor` instead of `id` in `Section`...
```suggestion
<Section anchor="about">
```